src/styles/main.scss

```diff
@@ -1,3 +1,28 @@
-body {
-  margin: 0;
+@import "./utils/variables";
+@import "./utils/_functions";
+@import "./block/reset.scss";
+@import "./block/header.scss";
+@import "./block/star.scss";
+@import "./block/cards.scss";
+
+.main {
+  &__wrapper {
+    max-width: getWrapperWidth($card-width, $card-quantity, $card-right-gap);
+    display: flex;
+    flex-wrap: wrap;
+    padding-top: 50px;
+    margin: 0 auto;
+  }
+
+  &__card--pos_dist {
+    margin-right: $card-right-gap;
+  }
+
+  &__wrapper--pos_dist &__card--pos_dist:nth-last-child(-n + 4) {
```

Comment:
Rename classes, don't use shortened words.
Also, nest it under wrapper selector.